samples/open-telemetry/application-insights/Core_8/Endpoint/Program.cs

```diff
@@ -60,6 +60,10 @@ static async Task Main(string[] args)
         endpointConfiguration.EnableOpenTelemetry();
         #endregion
 
+        #region enable-metrics-shim
+        endpointConfiguration.EnableMetricsShim();
```

Comment:
Shouldn't this be auto-enabled if you're opting into Otel? Otherwise your endpointconfiguration becomes cluttered with the exact same intent.

Reply:
That was the original plan but hooking up to the retries requires access to the endpoint configuration and I couldn't find a workaround :(

Reply:
see https://github.com/Particular/docs.particular.net/pull/6452/files#diff-4efc07ff98883f735fa775086dce63c4935090f74a55b087de83d7eba86f9247R12

Reply:
Let's discuss wednesday ;)

Reply:
@lailabougria I was able to auto-enable it, please take another look